cms/static/js/certificates/spec/views/certificate_editor_spec.js

```diff
@@ -239,7 +239,7 @@ function(_, Course, CertificateModel, SignatoryModel, CertificatesCollection, Ce
                 });
 
                 setValuesToInputs(this.view, {
-                    inputSignatoryTitle: 'New Signatory title longer than 40 characters on one line'
+                    inputSignatoryTitle: 'New Signatory title longer than 106 characters on one lineNew Signatory title longer than 106 characters on one line'
```

Comment:
It's just text character input for the test -- otherwise we can use a 106-character sentence :)